app/services/rbac_api.rb

```diff
@@ -1,32 +1,63 @@
 # frozen_string_literal: true
 
-# This class is meant to handle calls to the RBAC light API
+require 'insights-rbac-api-client'
+
+# This class is meant to handle calls to the RBAC API
 class RbacApi
+  COMPLIANCE_FULL_ACCESS = 'compliance:*:*'
+  APPS = 'compliance,inventory'
+  ANY = '*'
+
   def initialize(b64_identity)
-    @url = URI.parse("#{URI.parse(Settings.rbac_url)}"\
-                     "#{Settings.path_prefix}/rbac/v1/access/")
+    @client = RBACApiClient::AccessApi.new
     @b64_identity = b64_identity
   end
 
-  def check_user
+  # rubocop:disable Metrics/MethodLength
+  def load_users_permissions
```

Comment:
```suggestion
  def load_user_permissions
```
Also should it not be a constant below, if you extracted it?